src/cloud-resources/src/crd/materialize.rs

```diff
@@ -127,7 +129,12 @@ pub mod v1alpha1 {
         #[serde(default)]
         pub in_place_rollout: bool,
         // The name of a secret containing metadata_backend_url and persist_backend_url.
+        // It may also contain external_login_password_mz_system, which will be used as
+        // the password for the mz_system user if authenticator_kind is Password.
         pub backend_secret_name: String,
+        // How to authenticate with Materialize. Valid options are Password and None.
+        // If set to Password, the backend secret must contain external_login_password_mz_system.
+        pub authenticator_kind: AuthenticatorKind,
```

Comment:
does this work for existing environments without the `#[serde(default)]` annotation?